cis_v110/section_3.sp

```diff
@@ -39,7 +42,7 @@ control "cis_v110_3_1" {
     cis_item_id = "3.1"
     cis_level   = "1"
     cis_type    = "automated"
-    service     = "audit"
+    service     = "OCI/Audit"
```

Comment:
I think the query used in this control is actually under the identity service, as the table name in the query is `oci_identity_tenancy`, should the query be updated to be under the `identity` folder and be renamed, and then this service would be `OCI/Identity`?

Reply:
@cbruno10 table still uses the [audit API of OCI](https://github.com/turbot/steampipe-plugin-oci/blob/3b9d373c652e2b232765c7a31023f7544f112809/oci/table_oci_identity_tenancy.go#L6) along with others, makes sense to move the query to Identity as `identity_audit_log_retention_period_365_days.sql`

Reply:
@rajlearner17 For query name, would `identity_tenancy_audit_log_retention_period_365_days.sql` work (main change being including the resource type as well)?